src/components/scopedVariables/types.tsx

```diff
@@ -5,6 +5,16 @@ export enum ReadFileAs {
     ARRAY_BUFFER = 'arrayBuffer',
 }
 
+export enum FileView {
+    YAML = 'yaml',
+    SAVED = 'saved',
+}
+
+export interface SavedVariablesViewI {
+    scopedVariables: any
+    setScopedVariables: (variables: any) => void
```

Comment:
Variables type?

Reply:
Right now the schema is not fixed, will update it on confirmation.